build/bazel/remote/execution/v2/remote_execution.proto

```diff
@@ -152,6 +239,25 @@ service ActionCache {
     option (google.api.http) = { get: "/v2/{instance_name=**}/actionResults/{action_digest.hash}/{action_digest.size_bytes}" };
   }
 
+  // Retrieves a set of potential matches for a weak fingerprint. If a
+  // specific result matches the client's filesystem, the client uses the
+  // related selector to retrieve the build outputs using
```

Comment:
>selector [](start = 13, length = 8)

Should we describe what we mean by selector here?

Reply:
Fixed.

---
In reply to: [458989637](https://github.com/erikma/remote-apis/pull/1#discussion_r458989637) [](ancestors = 458989637)